### PR TITLE
Creating rc-slider typescript definition file

### PR DESCRIPTION
**Pull request by mantasmarcinkus**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by mantasmarcinkus** [comment]
Can this be reviewed? 🥇

File: rc-slider/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+ {
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (minestarks) [inline rc-slider/tsconfig.json]: New packages should have ```strictNullChecks``` on.

File: rc-slider/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    export interface HandleProps extends CommonApiProps {
+        /**
+         * Class name
+         */
+        className: string;
+        /**
+         * Styling if true, then bottom: {offset} else left: {offset}
+         * @default False
+         */
+        verical: boolean;
```
Comment (minestarks) [inline rc-slider/index.d.ts]: Is this a typo?

File: rc-slider/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        tipTransitionName?: string;
+
+        /**
+         * @deprecated in version ^6.0.0. Use rc-tooltip
+         * Tooltip formatter
+         */
+        tipFormatter?: ((value: any) => any | undefined) | null;
+    }
+
+    export interface SliderProps extends CommonApiProps {
```
Comment (minestarks) [inline rc-slider/index.d.ts]: You shouldn't need to export these interfaces, do you?

**Review by minestarks — CHANGES_REQUESTED** [review]

**Pull request merged**